learn/what_is_meilisearch/language.md

```diff
@@ -1,42 +1,43 @@
 # Language
 
-**Meilisearch is multilingual**, featuring optimized support for:
+Meilisearch is multilingual, featuring optimized support for:
 
 - **Any language that uses whitespace to separate words**
-- **Chinese** (through [Jieba](https://github.com/messense/jieba-rs))
-- **Japanese** (through [Lindera](https://github.com/lindera-morphology/lindera))
+- **Chinese**
```

Comment:
I think we should "un-bold" lines 5-8

Reply:
I went through the same thought process, unbolded them, thought it looked less good, then put the bold back. Anyway I agree that it doesn't match our style guide so consider this change made ✅